source/includes/options-mongosqld.yaml

```diff
@@ -135,10 +135,10 @@ name: mongo-versionCompatibility
 directive: option
 args: <version-number>
 description: |
-  Specifies that {{program}} must only use features supported by the
-  specified version of MongoDB. Only necessary when used with
-  replica sets in which members use different MongoDB versions.
-  The minimum allowable MongoDB version is ``3.2``.
+  Specifies that :program:`mongosqld` must only use features that the
+  specified version of MongoDB supports. Only necessary when used with
```

Comment:
I'm not a huge fan of the redundant "specifies/specified." It makes it seem like the user is providing two things. Maybe something like:

"Instructs/Restricts/Enforces :program:`mongosqld` to only use features that the specified version of MongoDB supports."
-or-
"The :program:`mongosqld` must only use features that the specified version of MongoDB supports."